packages/workbox-cli/src/lib/cleanup-stack-trace.js

```diff
@@ -0,0 +1,31 @@
+/**
```

Comment:
This might be overkill, but if we keep it, I'll add a test.

Reply:
@gauntface was in favor of the cleaner stack traces, so I've kept it and added a test for this module.